nix/sources.json

```diff
@@ -17,17 +17,17 @@
         "homepage": "https://input-output-hk.github.io/haskell.nix",
         "owner": "serokell",
         "repo": "haskell.nix",
-        "rev": "a4e105dcdb6d2b9d0627b8f0b893cec80485c14e",
-        "sha256": "144cfklhmqrq1kmvnmvcfmn819ijy4hr20whfazypxa72lcs3lz0",
+        "rev": "c427aeda2bc82fe4f8ff9354e70bdb4ecb5121b2",
+        "sha256": "1l2n46n5ym42ksyk7mvsh1a89ccdf4kc6k3vn4nvf22rvwm5v0w5",
         "type": "tarball",
-        "url": "https://github.com/serokell/haskell.nix/archive/a4e105dcdb6d2b9d0627b8f0b893cec80485c14e.tar.gz",
+        "url": "https://github.com/serokell/haskell.nix/archive/c427aeda2bc82fe4f8ff9354e70bdb4ecb5121b2.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "ligo": {
```

Comment:
AFAICT you updated to version `0.46.0` however the `docs/building.md` hasn't been updated to mention that this is the latest version used.